shell/testing/tester_main.cc

```diff
@@ -68,10 +68,10 @@ class TesterGPUSurfaceSoftware : public GPUSurfaceSoftware {
                            bool render_to_surface)
       : GPUSurfaceSoftware(delegate, render_to_surface) {}
 
-#if DISABLE_SOFTWARE_RASTER_CACHE
+#if SUPPORT_FRACTIONAL_TRANSLATION
```

Comment:
Might as well key this off of SUPPORT_FRACTIONAL_TRANSLATION, so we only need to change the location of the downloaded artifact and not the name too. I'll send a PR to revert the recipe change later